mirrord/config/src/util.rs

```diff
@@ -51,6 +51,99 @@ where
     type Generator = T::Generator;
 }
 
+impl<'de, T> Deserialize<'de> for ToggleableConfig<T>
```

Comment:
Can't we just ask `T: Deserialize`? or am I missing something?

Reply:
The untagged enum generates disgusting errors, if we clamp with only bool for toggle and the rest directly to T it fixes the error messages greatly

Reply:
Ah alright. Maybe put what you told me as a docs on the `ToggleableConfig`, so we know the why it is like this.